assisted-events-scrape/repositories/cluster_repository.py

```diff
@@ -0,0 +1,18 @@
+from assisted_service_client import rest
+
+
+class ClusterRepository:
+    def __init__(self, assisted_client):
+        self.client = assisted_client
+
+    def list_clusters(self):
+        return self.client.clusters_list()
+
+    def get_cluster_hosts(self, cluster_id: str) -> list:
+        hosts = []
+        try:
+            hosts = self.client.get_cluster_hosts(cluster_id=cluster_id)
+        except rest.ApiException as e:
+            if e.reason != "Not Found":
```

Comment:
please log even for "Not found". "the cluster has been deleted" or similar message

Reply:
This will be logged (and sent to sentry) by https://github.com/openshift-assisted/assisted-events-scrape/pull/49/files#diff-9bf5d2c8ddd304179959d273e721b6e244a544883b8139c41abb607828378018R47

Not sure is worth adding further info to this specific message? WDYT?

Reply:
we catch this exception